package.json

```diff
@@ -9,7 +9,7 @@
     "dev:build": "rimraf dist/development && cross-env NODE_ENV=development webpack",
     "prod:build": "rimraf dist/production && cross-env NODE_ENV=production webpack -p",
     "lint": "eslint ./",
-    "test": "mocha-phantomjs ./test/test.html",
+    "test": "yarn lint",
```

Comment:
Terminal commands can be chained by using `&&` between commands. For example: `terminal_command1 && terminal_command2`. This means that `terminal_command2` will only run when `terminal_command1` exits successfully.